src/bufferutils.js

```diff
@@ -174,8 +119,8 @@ module.exports = {
   readUInt64LE: readUInt64LE,
   readVarInt: readVarInt,
   reverse: require('buffer-reverse'),
-  varIntBuffer: varIntBuffer,
-  varIntSize: varIntSize,
+  varIntBuffer: varuint.encode,
+  varIntSize: varuint.encodingLength,
```

Comment:
I think they should be included for backward compatibility (at least while 3.x not be released).